.github/workflows/dockerhub_release.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: Build & Push Docker Image
+
+# Only builds the image if a release was created.
+# This guarantees the commit is tagged and on main branch.
+# (see https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases)
+
+# yamllint disable-line rule:truthy
+on:
+  release:
+
+jobs:
+  default:
+    # yamllint disable-line rule:line-length
+    uses: Hapag-Lloyd/Workflow-Templates/.github/workflows/docker_dockerhub_release_callable.yml@733afe1105fc85fafc25cddbc3c34b48c2c3d6fa
+    secrets: inherit
+    with:
+      # TODO insert correct value for image-name
+      # type: string
+      # required: true
+      # description: Name of the image. Usually it is sensible to set this to the repository name
+      image-name: "my-special-value"
+      # TODO insert correct value for push
+      # type: boolean
+      # required: false
+      # description: Flag to enable the workflow to actually push the built image to the registry
+      push: "my-special-value"
```

Comment:
not sure what happens here if this is not either true or false. Initially I would set this to `false` in any case to prevent pushing something to dockerhub from this template repo..

Reply:
Nothing we want, I guess. This workflow will be disabled. There is a init-workflow branch available which will fix everything.